tools/release/adot-operator-images-mirror/mirror.go

```diff
@@ -35,6 +37,11 @@ type GCRTagsResponse struct {
 	Tags []string `json:"tags"`
 }
 
+// TagsAllowlist contains information for the source and the allowed list of tags for that source
+type TagsAllowlist struct {
+	Tags []string `yaml:"tags"`
```

Comment:
I think this needs to go in the `Repository` struct so that allowed tags can be admitted at a per-repo level.

Reply:
Fixed